### PR TITLE
Flows and atoms for shrink cluster

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @nnDarshan @nthomas-redhat @shtripat please review

File: tendrl/node_agent/provisioner/gluster/plugins/gdeploy.py [hunk 1]
```diff
@@ -226,6 +228,70 @@ def shrink_gluster_cluster(self, host):
             return False
         return True
 
+    def stop_gluster_service(self, hosts):
+        self._reload_modules()
+        # hostnames = [NS.node_context.fqdn]
```
Comment (shtripat) [inline tendrl/node_agent/provisioner/gluster/plugins/gdeploy.py]: Remove the commented line

File: tendrl/node_agent/provisioner/gluster/plugins/gdeploy.py [hunk 2]
```diff
@@ -226,6 +228,70 @@ def shrink_gluster_cluster(self, host):
...
+                    },
+                    cluster_id=NS.tendrl_context.integration_id,
+                )
+            )
+            return False
+        return True
+
+    def stop_sds_service(self, hosts):
+        self._reload_modules()
+        # hostnames = [NS.node_context.fqdn]
```
Comment (shtripat) [inline tendrl/node_agent/provisioner/gluster/plugins/gdeploy.py]: Same as above

File: tendrl/node_agent/provisioner/gluster/plugins/gdeploy.py (hunk 1 above)
Comment (GowthamShanmugam) [inline tendrl/node_agent/provisioner/gluster/plugins/gdeploy.py]: i will remove it

**Pull request closed without merging**